pkg/console/util.go

```diff
@@ -1235,3 +1235,24 @@ func runCommand(cmd *exec.Cmd) error {
 	}
 	return err
 }
+
+// Get fully-qualified Hostname
+func GetFullHostname() (string, error) {
+	var (
+		hcmd      string
+		hostname string
+		out      []byte
+		err      error
+	)
+
+	// find node hostname
+	hcmd = `hostnamectl --static | tr -d '\r\n'`
+	out, err = exec.Command("/bin/sh", "-c", hcmd).Output()
+	if err != nil {
+		logrus.Errorf("failed to get hostname: %v", err)
+		return "", err
+	}
```

Comment:
We should use `CombinedOutput()` to capture both stdout and stderr. Also, `if err !=nil`, `out` should be logged too to capture the actual Linux outputs. In most cases, `err` only capture Go-specific errors. It will be great if we can revert back to using `os.Hostname()`, for portability.